### PR TITLE
closes #552 symbol tables german postcode

### LINKED ISSUE #552: Symbols: Deutsche Post "Zielcode"
https://de.wikipedia.org/wiki/Zielcode#Aussehen_und_Format

### PR DESCRIPTION
**Pull request by t-m-z**
;-) 

compile and test on emulator was successful

### CODE REVIEW

File: lib/widgets/searchstrings/searchstrings_de.dart [hunk 1]
```diff
@@ -447,6 +447,8 @@ const SEARCHSTRING_DE_SYMBOL_PLANET = '';
 const SEARCHSTRING_DE_SYMBOL_PLANETS =
     'himmelskoerper sonne mond sterne astologie astronomie merkur venus erde mars jupiter uranus neptun pluto asteroiden meteoriden gestirne';
 const SEARCHSTRING_DE_SYMBOL_POKEMON_UNOWN = '';
+const SEARCHSTRING_DE_SYMBOL_POSTCODE01247 = 'deutsche';
```
Comment (S-Man42) [inline lib/widgets/searchstrings/searchstrings_de.dart:450]: Many more searchstrings possible:  Zielcode, Anschriftencode, Postcode, Adresscode, Strichcode

File: lib/widgets/searchstrings/searchstrings_en.dart [hunk 2]
```diff
@@ -440,6 +440,8 @@ const SEARCHSTRING_EN_SYMBOL_PLANET = '';
 const SEARCHSTRING_EN_SYMBOL_PLANETS =
     'celestialbodies celestianbody heavenlybodies heavenlybody planets astronomy astrology stars meteroids asteroids sun moon earth saturn neptune jupiter pluto uranus mars venus mercury';
 const SEARCHSTRING_EN_SYMBOL_POKEMON_UNOWN = '';
+const SEARCHSTRING_EN_SYMBOL_POSTCODE01247 = 'german';
```
Comment (S-Man42) [inline lib/widgets/searchstrings/searchstrings_en.dart:443]: barcode

File: lib/widgets/searchstrings/searchstrings_fr.dart [hunk 3]
```diff
@@ -428,6 +428,8 @@ const SEARCHSTRING_FR_SYMBOL_PLANET = '';
 const SEARCHSTRING_FR_SYMBOL_PLANETS =
     'corpscelestes astre planetes mercure venus terre mars jupiter saturne uranus neptune pluto soleil lune asteroide meteorite astrologie astronomie etoile ';
 const SEARCHSTRING_FR_SYMBOL_POKEMON_UNOWN = '';
+const SEARCHSTRING_FR_SYMBOL_POSTCODE01247 = 'allemmande';
```
Comment (S-Man42) [inline lib/widgets/searchstrings/searchstrings_fr.dart:431]: one time you write it with one "m", another "mm".

File: lib/widgets/searchstrings/searchstrings_en.dart (hunk 2 above)
Comment (t-m-z) [reply lib/widgets/searchstrings/searchstrings_en.dart:443]: barcode is already part of search for these symbols

File: lib/widgets/searchstrings/searchstrings_de.dart (hunk 1 above)
Comment (t-m-z) [reply lib/widgets/searchstrings/searchstrings_de.dart:450]: done

**Pull request merged**